examples/media/schema.json

```diff
@@ -281,13 +277,22 @@
                   ]
                 }
               }
-            }
+            },
+            "scopesNeeded": [
+              "MEDIA_WRITE"
+            ]
           },
           "get": {
-            "parameters": {}
+            "parameters": {},
+            "scopesNeeded": [
+              "MEDIA_READ"
+            ]
           },
           "subscribe": {
```

Comment:
Differentiation between _subscribe_ and _get_ is not clear. We should talk about it.

Reply:
Generally the same.. Just it might be, a service does not offer subscription... It is the replacement for systemTriggeredEvents ;)

Reply:
I do understand your intention. But there is a general minimum feature set defined for the protocol. This includes support of subscriptions.

Reply:
I still vote for a separate `subscription` entry. Just in case we need to differentiate

Reply:
Do we want to show the similarities or differences between subscriptions and GET requests in the schema?

Reply:
there might be differences.. so I would like to express that, also there might be a service that only supports GET, no subscribe, so by absences of the `subscribe` key this would be expressed